### DISCUSSION TITLE
317

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#181] [Comments]
# 317

Intro Pull Request를 기다리다가, merge가 되었다고 착각하고 커밋 했던 브랜치를 삭제해 버렸습니다.. 로컬과 remote 모두에서 삭제 했기 때문에 원래대로라면 데이터를 날려먹은게 맞지만, 다행히도 복구하는 방법이 있습니다. branch 생성 및 삭제 같은 상황을 만들기 위해 branch를 생성 해서 커밋 한 후 삭제 하겠습니다. 이미 branch가 삭제되어 복구가 필요한 분은 아래로 스크롤을 내려 삭제한 branch 복구 를 확인하시면 됩니다. branch 생성 및 commit & push dev 라는 branch를 만들어서, gitbranch.txt 파일을 추가해 commit 및 push 까지 마친 상태입니다. 이 상태에서는 dev에 커밋이 되었지만, 해당 사항이 master에 까지..

https://shanepark.tistory.com/317

**Comment by kim-src** [comment]
옴마나!!!!!!! 진짜 복구됐어요!!!!!!! ㅜㅜ 감사합니다... 이제 잘게요..
